lib/web_apis/api_importer.es6.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+foam.CLASS({
+  name: 'ApiImporter',
+  package: 'org.chromium.apis.web',
+  documentation: `API Importer is a class that handles importing browserAPIs
+    from webCatalog object to browserAPI DAO.`,
+  requires: [
+    'org.chromium.apis.web.BrowserAPI',
+    'foam.mlang.ExpressionsSingleton',
+    'foam.dao.EasyDAO',
+  ],
+  properties: [
+    {
+      name: 'browserAPIs',
+      documentation: `This should be a server GCD DAO that contains
+        interface, API information for major browsers and versions
+        for recent 2 years. It is implemented as a MDAO for now.`,
+      factory: function() {
+        let browserApiDao = this.EasyDAO.create({
+          name: 'BrowserAPIDAO',
+          of: this.BrowserAPI,
+          daoType: 'MDAO',
+        });
+        browserApiDao.addPropertyIndex(this.BrowserAPI.BROWSER_KEY);
```

Comment:
No need to change; this is fine for now.

But a controller should probably determine the make-up of this DAO eventually. That is, it will be passed in when instances of this class are constructed, and indexes that ensure acceptable performance on the analysis the controller wishes to do will be configured here.